### ISSUE TITLE
Model Data Validation

### ISSUE THREAD

**Issue by VBoB13** [VBoB13/TeachAdmin#3]
### Models

I need to implement basic validators to make sure the user's input data gets validated properly before being stored in the database. Therefore, in whatever way it needs to be implemented (i.e. field validators and/or model validators), the core validations must be done before the alpha release.
Some models that definitely needs to have validators implemented are:

## Scores
- [x] ExamScore
- [x] AssignmentScore
- [x] LessonTestScore
- [x] HomeworkScore

**Comment by piyushpradhan** [comment]
Can I work on this ? or is this issue resolved ?

**Comment by VBoB13** [comment]
**I'm so sorry** I didn't reply sooner! I'm my honest opinion, I never thought anyone would want to contribute to this project at this early stage. _I'm honored that you want to!_
This is kind of my "portfolio project", but at the same time my one best idea and purpose I've got, so it's going to be _"my baby" that I will keep improving._
Now to the point: I'm learning a lot these days as I've finally started to get there hang of them basics, and I just read about, learned and implemented some of them major validators in (I think?) all of the score models. I'm in bed right now and lay to check into details, so I will check this up tomorrow and update you on further issues that could need your help as well!
On the same note, I also don't expect anything. You do what you can, when you can, in the way you deemed best. So never feel pressure 👍

**Comment by piyushpradhan** [comment]
Oh okay, no problem. Actually I'm just starting out with Open Source so, I'm just finding projects that I can contribute to. I'd love to contribute to this project, so if there's anything I can do, just let me know, I'll try my best to solve that issue.

**Comment by VBoB13** [comment]
Well, let me say **Welcome!** to you my kind fellow programmer 🎉
Today, I will work a little more on the project and I will most likely hit you up with something that you could do shortly~
However, since I don't know what exactly you're capable of right now, it'll be hard to assign anything per say, but you're always free to "claim" an issue you feel comfortable with 😄
Feel free to throw up some ideas along the way if you ever notice something that could be implemented and you believe improves the project.
In a nutshell - if you have any questions at all; feel free to ask at any time~ I will try my best to explain to/elaborate with you.

Best of luck!
// Richard
